src/components/dashboard/Dashboard.tsx

```diff
@@ -1,27 +1,181 @@
+
+  console.log(currDate);
+
+  if (currDate < openDate) {
+    return `Application opens on ${openDate.toLocaleString(DateTime.DATETIME_FULL)}`
+  } 
+  if (currDate > closeDate) {
+    return `Applications closed on ${closeDate.toLocaleString(DateTime.DATETIME_FULL)}`
+  }
+  return `Accepting applications until ${closeDate.toLocaleString(DateTime.DATETIME_FULL)}`
+}
 
 const Dashboard: React.FC = () => {
   const { user } = useAuth();
+  const currentHexathon = useCurrentHexathon();
 
-  const [{ data: profile, loading, error }] = useAxios(
+  const [{ data: profile, loading: profileLoading, error: profileError }] = useAxios(
     `https://users.api.hexlabs.org/users/${user?.uid}`
   );
 
-  if (loading || error) {
-    return <Loading />;
-  }
+  const [{ data: branches, loading: branchesLoading, error: branchesError }] = useAxios(
+    `https://registration.api.hexlabs.org/branches/?hexathon=${currentHexathon._id}`
+  );
+
+  const [{data: application, loading: applicationLoading, error: applicationError}] = useAxios({
+    url: 'https://registration.api.hexlabs.org/applications/',
+    method: 'GET',
+    params: {
+      hexathon: currentHexathon._id,
+      userId: user?.uid
+    }
+  })
 
-  if (profile.permissions?.admin) {
-    return <AdminDashboard profile={profile} />;
+  if (profileLoading || profileError 
+    || branchesLoading || branchesError ) {
+    return <Loading />;
   }
 
-  return <ParticipantDashboard profile={profile} />;
+  return (
+    <Flex
+      flexDir="column"
+      padding={{ base: "0 0 16px", md: "32px 48px" }}
+      margin="auto"
+      maxWidth="1000px"
+    >
+      <Flex
+        flexDir={{ base: "column", md: "row" }}
+        bgGradient={{
+          base: "linear(to-b, #33c2ff, #7b69ec)",
+          md: "linear(to-r, #33c2ff, #7b69ec)",
+        }}
+        boxShadow="rgba(0, 0, 0, 0.24) 0px 3px 8px"
+        alignItems="center"
+        justifyContent="space-around"
+      >
+        <Box
+          color="white"
+          paddingY="32px"
+          paddingLeft={{ base: "16px", md: "32px" }}
+          paddingRight={{ base: "16px", md: "0px" }}
+        >
+          <Heading size="xl" marginBottom="15px">
+            Welcome Back {profile.name?.first}!
+          </Heading>
+          <Text>
+            We’re happy to see you here! We’re currently running our {currentHexathon.name} and we’d
+            love to see you there!
+          </Text>
+          <HStack
+            maxWidth={{ base: "500px", md: "400px" }}
+            justifyContent="space-around"
+            marginTop={{ base: 4, md: 8 }}
+            spacing={4}
+          >
+            <Button width="45%" color="#7B69EC" borderWidth="1px" borderColor="#7B69EC">
+              Apply
+            </Button>
+            <Button width="45%" bgColor="#7B69EC">
+              Details
+            </Button>
+          </HStack>
+        </Box>
+        {application.confirmed ? (
```

Comment:
So the api for checking the confirmed status for an application has changed. If you look at the [docs site](https://docs.hexlabs.org/#/registration/get_applications__id_), you'll see, but there is a new field called `status` that you will have to check is equal to `CONFIRMED`. You can see the PR that changed this here https://github.com/HackGT/api/pull/75